introduction.md

```diff
@@ -0,0 +1,7 @@
+git clone https://github.com/username/dojo_rules.git
```

Comment:
Done. Is this all correct?

On Wed, May 25, 2016 at 2:04 PM, Code School Kiddo <notifications@github.com
> wrote:

> In introduction.md
> <https://github.com/deadlyvipers/dojo_rules/pull/10149#discussion_r64621564>
> :
>
> > @@ -0,0 +1,7 @@
> > +git clone https://github.com/username/dojo_rules.git
>
> Looks good! Could you also please mention your favorite Code School path
> in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or
> Electives.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/deadlyvipers/dojo_rules/pull/10149/files/5be2496b7dad53059ceddcc7992366f5214e8b55#r64621564>
>